Makefile

```diff
@@ -63,13 +58,19 @@ LINKFLAGS ?=
 ifeq ($(TYPE),)
 override LINKFLAGS += -X github.com/cockroachdb/cockroach/pkg/build.typ=development
 else ifeq ($(TYPE),release)
-override LINKFLAGS += -s -w -X github.com/cockroachdb/cockroach/pkg/build.typ=release
+# We use a custom toolchain to target old Linux and glibc versions. However,
+# this toolchain's libstdc++ version is quite recent and must be statically
+# linked to avoid depending on the target's available libstdc++.
+export CC  = /x-tools/x86_64-unknown-linux-gnu/bin/x86_64-unknown-linux-gnu-gcc
+export CXX = /x-tools/x86_64-unknown-linux-gnu/bin/x86_64-unknown-linux-gnu-g++
+override LINKFLAGS += -s -w -extldflags '\''-static-libgcc -static-libstdc++'\'' -X github.com/cockroachdb/cockroach/pkg/build.typ=release
+override GOFLAGS += -installsuffix release
 else ifeq ($(TYPE),musl)
 # This tag disables jemalloc profiling. See https://github.com/jemalloc/jemalloc/issues/585.
 override TAGS += musl
 export CC  = /x-tools/x86_64-unknown-linux-musl/bin/x86_64-unknown-linux-musl-gcc
 export CXX = /x-tools/x86_64-unknown-linux-musl/bin/x86_64-unknown-linux-musl-g++
-override LINKFLAGS += -extldflags -static -X github.com/cockroachdb/cockroach/pkg/build.typ=release-musl
+override LINKFLAGS += -s -w -extldflags '\''-static'\'' -X github.com/cockroachdb/cockroach/pkg/build.typ=release-musl
```

Comment:
Lots of precedent here for this kind of quote literal, and double quotes have different meaning wrt expansion.